src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSize.cs

```diff
@@ -0,0 +1,105 @@
+using System;
```

Comment:
All code files must have the license header.

```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Reply:
Added [here](https://github.com/dotnet/installer/pull/17233/files#diff-ce0a9d7a18f02288a0510a8ec86e3a5f387ad96613e0296f434de93f02c18da7R1).